remo/events/admin.py

```diff
@@ -11,31 +10,13 @@ class AttendanceInline(admin.StackedInline):
     model = Attendance
 
 
-class EventGoalAdmin(ExportMixin, admin.ModelAdmin):
-    """EventGoal Inline."""
-    model = EventGoal
-
-
 class EventResource(resources.ModelResource):
-    event_goals = fields.Field()
     event_categories = fields.Field()
     event_metrics = fields.Field()
 
     class Meta:
         model = Event
-        exclude = ('id', 'categories', 'goals', 'metrics',)
-
-    def dehydrate_event_goals(self, event):
-        if event.goals.all().exists():
-            goals = ', '.join(x.name for x in event.goals.all())
-            return goals
-        return ''
-
-    def dehydrate_event_categories(self, event):
```

Comment:
We are not removing the ``event_categories`` field. We might want to keep this ``dehydrate_`` method unchanged.